csrc/map/tensor_map.cpp

```diff
@@ -0,0 +1,49 @@
+#include <torch/extension.h>
```

Comment:
For the new files, you'll need to add the license at the top of the file to pass the formatting check.